cli/src/lib/compiler/compileLibrary.js

```diff
@@ -81,6 +81,8 @@ const compileLibrary = async ({ config, paths, mode, watch }) => {
             process.exit(1)
         }
     } else {
+        fs.ensureDirSync(outDir)
```

Comment:
Is there a reason not to remove the existing directory before running the compiler in watch mode?

Reply:
Yes, Webpack Dev Server crashes if the directory disappears and needs to be restarted. Just overwriting the files works however, but we need to ensure that the dir is there on first run.

Reply:
Meaning webpack devserver somewhere else in the monorepo?  Unfortunate, but OK for now 👌

Reply:
Yeah, for example when I run Storybook for ui-core and the `build` folder disappears, Storybook crashes, so deleting it on every rebuild is a bit annoying.